src/client/components/ProfileBlueBar/ProfileBlueBar.component.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import backArrow from '../../assets/images/profile_back_arrow.png';
+import './ProfileBlueBar.styles.css';
+
+export default function ProfileBlueBar() {
+  return (
+    <div className="profile-bar-container">
+      <div className="profile-bar-content">
+        <h2 className="profile-bar-title">GuideIT</h2>
+        <h4 className="profile-bar-name">Hello, Anne</h4>
+      </div>
+      <div className="profile-bar-footer">
+        <button type="button" className="profile-bar-log-out">
+          Logout
+        </button>
+        <div className="profile-bar-footer-container">
+          <img src={backArrow} alt="back arrow" />
+          <a href="/home" className="profile-bar-back">
```

Comment:
it looks great, I am just wondering if the `/home`  shouldn't be a prop so you can pass it in the app.js? I am just wondering maybe I am wrong :)

Reply:
@abedaarabi /home is not a prop, its just a path.